cpp/arcticdb/version/python_bindings.cpp

```diff
@@ -138,7 +138,30 @@ void register_bindings(py::module &version, py::exception<arcticdb::ArcticExcept
             .value("SPECIFIC", VersionRequestType::SPECIFIC)
             .value("LATEST", VersionRequestType::LATEST);
 
-    py::class_<DataError, std::shared_ptr<DataError>>(version, "DataError")
+    py::class_<DataError, std::shared_ptr<DataError>>(version, "DataError", R"pbdoc(
+        Return value for batch methods which fail in some way.
+
+        Attributes
+        ----------
+        symbol: str
+            Read or modified symbol.
+        version_request_type: Optional[VersionRequestType]
+            For operations that support as_of, the type of version query provided. `None` otherwise.
+        version_request_data: Optional[Union[str, int]]
+            For operations that support as_of, the value provided in the version query:
+                None - Operation does not support as_of, or latest version was requested
+                str - The name of the snapshot provided to as_of
+                int - The specific version requested if version_request_type == VersionRequestType::SPECIFIC, or
+                      nanoseconds since epoch if version_request_type == VersionRequestType::TIMESTAMP
+        error_code: Optional[ErrorCode]
```

Comment:
Can we link to the error code docs here?

Reply:
Yeh I think so. I guess it's a bit odd having a URL in the docstring but the alternative is actually documenting `ErrorCode` which would be duplciating the work we've already done I think

Reply:
Yup sure that seems sensible